testing/test_assertion.py

```diff
@@ -338,15 +338,13 @@ def test_different_types(self) -> None:
     def test_summary(self) -> None:
         lines = callequal([0, 1], [0, 2])
         assert lines is not None
-        summary = lines[0]
-        assert len(summary) < 65
+        assert len(lines) == 3
 
     def test_text_diff(self) -> None:
-        assert callequal("spam", "eggs") == [
```

Comment:
as far as I can tell this greatly diminishes the value if intentionally concrete tests

please explain the rationale

Reply:
hey yeah sorry, i am going to post a proper description, i thought i was merging in my own main branch no the pytest one.